main.cpp

```diff
@@ -83,43 +105,91 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
+ */
+float bodyMassIndex(float height = 175.f, float weight = 75.f)
+ {
+    ignoreUnused(height, weight);
+    return {};
+ }
 
 /*
  2)
- */
+ */
+double averageMathScore(double score, int studentID)
+{
+    ignoreUnused(score, studentID);
+    return {};
+}
 
 /*
  3)
- */
+ */
+void setGrade(int mark = 60)
+{
+    ignoreUnused(mark);
+}
 
 /*
  4)
  */
+bool insectFlies(bool hasWings = true)
+{
+    ignoreUnused(hasWings);
+    return {};
+}
 
 /*
  5)
- */
+ */
+int averageWaitTime(char hospitalCode, char clinicCode, bool isAdult = true)
+{
+    ignoreUnused(hospitalCode, clinicCode, isAdult);
+    return {};
+}
 
 /*
  6)
- */
+ */
+void moveChessPiece(char pieceCode = 'P', char coordLetter = 'A', int coordNum = 7)
+{
+    ignoreUnused(pieceCode, coordLetter, coordNum);
+}
 
 /*
  7)
- */
+ */
+bool finishedRace(float distTravelled)
```

Comment:
This function name does not contain a verb/action
Function names describe work that is performed, that means they need to have verbs in their name
- [ ] revise